### ISSUE TITLE
Translation

### ISSUE THREAD

**Issue by annaproxy** [Sasafrass/straattaal#54]
Generated words are "in Dutch" and are probably most fun for Dutch speakers. 
As a (parody of a) user, I am confused: should I be entering a definition in English?
In the future, we could provide the app in English/Dutch, depending on the language of the current model / user preference.

**Comment by Sasafrass** [comment]
Yes, this is a good addition. Also allows for some experimental stuff with detecting someone's locale and changing the application accordingly.
